exercises/largest-series-product/LargestSeriesProduct.example

```diff
@@ -0,0 +1,53 @@
+module LargestSeriesProduct exposing (largestProduct)
+
+import String
+
+
+largestProduct : Int -> String -> Maybe Int
+largestProduct length series =
+    if length < 0 then
+        Nothing
+    else if length == 0 then
+        Just 1
+    else
+        String.toList series
+            |> List.map (String.fromChar >> String.toInt)
```

Comment:
That's really really tempting, but unfortunately we're supposed to return `Nothing` if there is a non numeric char in the series, so we can't just discard them. So this would require a pre-validation step (which I'm not totally opposed to).

I will at least take the `String.split ""` simplification. :)